src/Stratis.Bitcoin.Features.Wallet/Broadcasting/FullNodeBroadcasterManager.cs

```diff
@@ -37,7 +37,7 @@ public FullNodeBroadcasterManager(IConnectionManager connectionManager, IMempool
                 return false;
             }
 
-            this.PropagateTransactionToPeers(transaction);
+            this.PropagateTransactionToPeers(transaction, true);
```

Comment:
I had suggested removing the boolean because it's going to be always true, but I'll leave this in for now.